src/libraries/Microsoft.PowerFx.Core/Parser/ParseNamedFormulasAndUDFResult.cs

```diff
@@ -0,0 +1,32 @@
+﻿// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+using System.Collections.Generic;
+using System.Linq;
+using Microsoft.PowerFx.Core.Errors;
+
+namespace Microsoft.PowerFx.Core.Parser
+{
+    internal class ParseUserDefinitionResult
```

Comment:
>internal [](http://example.com/codeflow?start=4&length=8)

We should expect that this class will soon become Public  (like ParseResult). 
It's ok for a public class to have internal methods.